scripts/patterns-reorg/redirects.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
This page is auto-generated.

We may want the data in alternative formats for future uses like setting up redirects in closed source. We can easily re-generate the file however we want.